src/stream/stream_backend.rs

```diff
@@ -37,10 +41,14 @@ fn check_endpoints(
         return Err(SimpleError::new("Endpoints are empty".to_string()));
     }
 
+    let is_custom_webrtc = (endpoints[0].scheme() == "stun")
+        || (endpoints[0].scheme() == "turn")
+        || (endpoints[0].scheme() == "ws");
```

Comment:
ws stands for websocket and for not webrtc, is this correct ? should we use "webrtc" ?

Reply:
You can also do != webrtc